packages/teleport/src/lib/DesktopAccess/codec.test.ts

```diff
@@ -0,0 +1,131 @@
+import Codec, { MessageType, ButtonState, MouseButton } from './codec';
+
+function getRandomInt(min, max) {
```

Comment:
Technically this is not an RDP codec, but a codec for our custom desktop protocol ([RFD 0037](https://github.com/gravitational/teleport/blob/master/rfd/0037-desktop-access-protocol.md)). RDP is used on the backend but that may not hold true for all instances in the future, such as when we add support for linux/macos.

I propose we name our protocol Teleport Desktop Protocol aka `tdp`. cc @awly since it's your brain-child.